src/morphAttrs.js

```diff
@@ -16,9 +16,15 @@ export default function morphAttrs(fromNode, toNode) {
         attrValue = attr.value;
 
         if (attrNamespaceURI) {
-            attrName = attr.localName || attrName;
```

Comment:
We use `|| attrName` because IE doesn't seem to support `.localName` [see](https://developer.mozilla.org/en-US/docs/Web/API/Attr/localName)

Reply:
Thanks for your feedback, very appreciated.
About which version of IE are we speaking? (IE9 supports localName, [see](http://help.dottoro.com/ljoiklkm.php)).
[Microsoft officially stopped to support old IE versions until version 11](https://www.microsoft.com/en-in/WindowsForBusiness/End-of-IE-support)

My point is that the `setAttributeNS` API is called with the wrong argument and I provided the official W3C specifications. May I ask you to study them yourself, look up all the technical definitions in the official DOM API glossary and then decide for yourself if I'm right or wrong.